index.js

```diff
@@ -3,5 +3,6 @@ const app = express()
 
 app.get('/', (req, res)=>{
     res.send('Hello world')
+    
 })
 app.listen(5500, () => console.log('listening on port 5500...'))
```

Comment:
You should add an empty line after line 8 to remove this lint error

Reply:
changes made.